### PR TITLE
Feat/arbitrary repo code reviews

### PR DESCRIPTION
**Pull request by yumoL**
### Short description

#517 Allow the teacher to add an arbitrary repository (which is not in the current course instance) for code review

### DoD

I have:
- [x] added actual code.
- [x] produced clean code that passes ESLint.
- [x] code that actually does what it should.
- [ ] documented the code or added documentation to the wiki.
- [x] added or modified test(s).
- [x] test(s) that actually pass.

### CODE REVIEW

File: backend/server/controllers/codeReviews.js [hunk 1]
```diff
@@ -23,15 +23,15 @@ module.exports = {
...
           return null // This will lead to rejection later.
         }
         allStudentInstancesIds.push(codeReview.reviewer)
-        allStudentInstancesIds.push(codeReview.toReview)
         return {
           studentInstanceId: codeReview.reviewer,
           reviewNumber: req.body.reviewNumber,
-          toReview: codeReview.toReview
+          toReview: Number.isInteger(codeReview.toReview) ? codeReview.toReview : null,
+          repoToReview: Number.isInteger(codeReview) ? null : codeReview.repoToReview
```
Comment (hisahi) [inline backend/server/controllers/codeReviews.js]: `Number.isInteger(codeReview)` is never true

Comment (yumoL) [reply backend/server/controllers/codeReviews.js]: Sorry for my carelessness, it should be `Number.isInteger(codeReview.toReview)`

**Comment by yumoL** [comment]
> A bunch of errors when loading the page:
> kuva

I may need some time to figure out the problem. How did you get this bug?

**Comment by hisahi** [comment]
I tried to open the "edit code reviews" page on the test DB (with latest migrations).

**Comment by yumoL** [comment]
> I tried to open the "edit code reviews" page on the test DB (with latest migrations).

These errors happen when none of the code review rounds has been selected. I tried to fix it, please have a check.

**Pull request merged**